src/test/scala/com/rawlabs/das/mock/DASMockTestApp.scala

```diff
@@ -12,12 +12,12 @@
 
 package com.rawlabs.das.mock
 
+import org.scalatest.funsuite.AnyFunSuite
+
 import com.rawlabs.das.server.DASServer
 
-// Run the main code with mock services
-object DASMockTestApp extends App {
+class DASMockTest extends AnyFunSuite {
 
-  // (Can be used to pass custom parameters/settings in the future?)
-  DASServer.main(Array())
+  test("Run the main code with mock services")(DASServer.main(Array()))
```

Comment:
Will this not hang the CI and leave it running?
Why make it a test? It's not testing anything (?)

Reply:
Yes it was left by accident as I run live tests like that. I rewrote it as a test as you used to have because `DASServer.main` wouldn't run within `test`.

It's missing the service declaration file. I'll switch it back to an `App` and add the service file too.